mocked_servers/grpc_metrics/Dockerfile

```diff
@@ -1,11 +1,17 @@
-FROM node:14
+FROM golang:1.17 AS mod
+WORKDIR $GOPATH/main
+COPY go.mod .
+COPY go.sum .
+RUN go env -w GOPROXY=direct
+RUN GO111MODULE=on go mod download
 
-WORKDIR /app
+FROM golang:1.17 as build
+COPY --from=mod $GOCACHE $GOCACHE
+COPY --from=mod $GOPATH/pkg/mod $GOPATH/pkg/mod
+WORKDIR $GOPATH/main
+COPY . .
+RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux go build -o=/bin/main .
 
-COPY *.js ./
-COPY package.json ./
-COPY metrichandler.proto ./
-
-RUN npm install
-
-CMD ["node", "metrics_handler_server.js"]
+FROM scratch
```

Comment:
Recommend `gcr.io/distroless/static` in case the lack of TLS certificates becomes an issue even in a mocked server in the future since the errors can be very mysterious

Reply:
Once gcr.io use has been verified, I can make the changes.